### PR TITLE
Adjust whitespace for better symmetry between Dockerfiles and cut image size in half by keeping fetch and compile in the same layer so the Python source goes away properly

### PR DESCRIPTION
**Pull request by tianon**

### CODE REVIEW

**Comment by tianon** [comment]
Before: ~163.2 MB
After: ~89.05 MB

File: 3.5/alpine/Dockerfile [hunk 1]
```diff
@@ -40,12 +42,16 @@ RUN apk add --no-cache --virtual .build-deps  \
...
-		done | sort -u) \
-	&& apk add --virtual .python-rundeps $rundeps \
+	&& runDeps="$( \
+		scanelf --needed --nobanner --recursive /usr/local \
+			| awk '{ gsub(/,/, "\n", $2); print "so:" $2 }' \
+			| sort -u \
+			| while read dep; do \
+				apk info --installed -q "$dep" && echo "$dep"; \
+			done \
+			| sort -u \
```
Comment (tianon) [inline 3.5/alpine/Dockerfile:50]: Is this second `sort -u` actually necessary?  Our `while` loop has just filtered the output of the first `sort`, so this one should be a no-op.

Comment (tianon) [inline 3.5/alpine/Dockerfile:50]: Nevermind, figured out something simpler that skips the loop. :smile:

**Comment by ncopa** [comment]
thanks! :+1:

**Comment by tianon** [comment]
Ah, I see -- in this case I think the `Dockerfile` maintenance burden is slightly more important since the only time things should be changing there is when we rebuild the image completely for a new version, so I think we're OK, but it's really good to know! :smile:  (Learning all kinds of cool stuff about `apk` through this exercise. :heart:)

**Pull request merged**